web/web.go

```diff
@@ -20,20 +19,57 @@ const (
 	defaultBind = ":8080"
 )
 
+type recoverer interface {
+	Capture(error, echo.Context)
+	Recover() echo.MiddlewareFunc
+}
+
 // Options contains a parameters for new Echo instance.
 type Options struct {
-	// TODO suggest change to echo.Logger
-	Logger    logger.Logger
-	Error     errors.Logic
+	Logger    echo.Logger
+	Error     recoverer
 	Debug     bool
 	Validator echo.Validator
 }
 
-// Context from echo.Context (for shadowing)
-type Context = echo.Context
+// Errors
+var (
+	ErrUnsupportedMediaType        = echo.ErrUnsupportedMediaType
+	ErrNotFound                    = echo.ErrNotFound
+	ErrUnauthorized                = echo.ErrUnauthorized
+	ErrForbidden                   = echo.ErrForbidden
+	ErrMethodNotAllowed            = echo.ErrMethodNotAllowed
+	ErrStatusRequestEntityTooLarge = echo.ErrStatusRequestEntityTooLarge
+	ErrValidatorNotRegistered      = echo.ErrValidatorNotRegistered
+	ErrRendererNotRegistered       = echo.ErrRendererNotRegistered
+	ErrInvalidRedirectCode         = echo.ErrInvalidRedirectCode
+	ErrCookieNotFound              = echo.ErrCookieNotFound
+)
+
+var (
+	// NewHTTPError creates a new HTTPError instance.
+	NewHTTPError = echo.NewHTTPError
+)
+
+type (
+	// Context from echo.Context (for shadowing)
+	Context = echo.Context
+
+	// HandlerFunc from echo.HandlerFunc (for shadowing)
+	HandlerFunc = echo.HandlerFunc
+
+	// MiddlewareFunc from echo.MiddlewareFunc (for shadowing)
+	MiddlewareFunc = echo.MiddlewareFunc
+
+	// Engine from echo.Echo (for shadowing)
+	Engine = echo.Echo
```

Comment:
mb we can extract interface?

Reply:
for now it's no sense

Reply:
why not?

Reply:
not need

Reply:
<img width="725" alt="maybe you wanna to talk about this" src="https://user-images.githubusercontent.com/888650/35923336-0182ddfe-0c31-11e8-9f95-93131bbb6550.png">